addOns/callhome/src/main/java/org/zaproxy/addon/callhome/InvalidServiceUrlException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2021 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.callhome;
+
+public class InvalidServiceUrlException extends Exception {
+
+    private static final long serialVersionUID = 1L;
+
+    public InvalidServiceUrlException(String url) {
+        super(url);
```

Comment:
Won't this mean the whole exception message is just the URL?

Reply:
I just ripped off the previous AutoUpdate exception :P